Program.cs

```diff
@@ -1,9 +1,13 @@
+using LinguaNews;
+using LinguaNews.Options;
+
 var builder = WebApplication.CreateBuilder(args);
 
 // Add services to the container.
 builder.Services.AddHttpClient();
 builder.Services.AddRazorPages();
-
+builder.Services.Configure<NewsApiOptions>(builder.Configuration.GetSection("NewsApiOptions"));
+builder.Services.AddScoped<INewsApiAiService, NewsApiAiService>();
```

Comment:
What is the purpose of this?  How is it used?
See notes above.
Is this used anywhere? I don't see any other references to it. If it's not used anywhere, it is unused code, which is clutter, which is technical debt, and should not be present in the main/master branch.